docker/run_webapp.sh

```diff
@@ -2,6 +2,12 @@
 
 # Prepares then runs the webapp.
 
+echo 'getting nvm/node/npm'
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+nvm install node
```

Comment:
This would be done every time we start a container, which I think we want to avoid. It should instead be done at build time. Probably in `set_up_ubuntu.sh` instead thus.